client/trino-client/src/main/java/io/trino/client/ClientSession.java

```diff
@@ -80,6 +81,7 @@ private ClientSession(
             Optional<String> user,
             Optional<String> sessionUser,
             Optional<String> authorizationUser,
+            Set<ClientSelectedRole> originalRoles,
```

Comment:
How do we pass original user info?

Reply:
Via sessionUser AFAIK, see https://github.com/trinodb/trino/blob/c8a173a6fe073d9fbfae36d9974a0098fb3538cf/client/trino-client/src/main/java/io/trino/client/StatementClientV1.java#L124